opencti-platform/opencti-front/src/private/components/arsenal/vulnerabilities/VulnerabilityKnowledge.js

```diff
@@ -64,6 +64,28 @@ class VulnerabilityKnowledgeComponent extends Component {
               />
             )}
           />
+          <Route
+            exact
+            path="/dashboard/arsenal/vulnerabilities/:vulnerabilityId/knowledge/targets"
+            render={(routeProps) => (
+              <EntityStixCoreRelationships
+                entityId={vulnerability.id}
+                relationshipTypes={['targets']}
+                isRelationReversed
+                entityLink={link}
+                allDirections={true}
+                stixCoreObjectTypes={[
+                  'Threat-Actor',
```

Comment:
According to the stix.ts file, it's missing the type Attack-Pattern.

Reply:
Maybe it's better to don't pass any entities ?

Reply:
May be you have a point about Attack-Pattern I'll check with Samuel, the list comes from him and we don't want anything else (to answer your second question)

Reply:
Fixed